test/resources/files/walletData.json

```diff
@@ -1,4 +1,13 @@
 {"address":"address",
 "masterKey":[
     
-]}
\ No newline at end of file
+],
+"importWallet": [
```

Comment:
masterKey array is already used to store the master key words and later on importing such master key, is there any reason why to create an additional array object (importWallet)?

Reply:
create this importWallet, based on the fact that later we will need a wallet with certain requirements eg. (with money), so it will be different from the master key.

On the other hand, if all the tests are executed in parallel in the master key array, the keys of a new wallet would be saved.

Reply:
great ;)
please follow the latest changes on master in order to handle it, since the walletData file won't be tracked anymore in the repository (for confidential reasons), instead it will be generated on execution time.
If you need that "importWallet" as a preexisting param, you can have it locally but untracked. Ideally, we could develop a service to generate wallets with specific conditions calling directly the available APIs (if possible)